CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,43 @@
+# Code of Conduct
+As contributors and maintainers of the DBOS-TS project, we pledge to respect everyone who contributes by posting issues, updating documentation, submitting pull requests, providing feedback in comments, and any other activities.
+
+## Our Standards
+Examples of behavior that contributes to creating a positive environment include:
+
+* Using welcoming and inclusive language.
+* Being respectful of differing viewpoints and experiences.
+* Gracefully accepting constructive criticism.
+* Focusing on what is best for the community.
+* Showing empathy towards other community members.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances.
+* Trolling, insulting/derogatory comments, and personal or political attacks.
+* Public or private harassment.
+* Publishing others' private information, such as physical or electronic addresses, without explicit permission.
+* Other conduct which could reasonably be considered inappropriate in a professional setting
+Our Responsibilities.
+* Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned with this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+
+## Scope
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
+
+## Enforcement
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at 
+peter.kraft@dbos.dev. All complaints will be reviewed and investigated promptly and fairly.
```

Comment:
We might want to give a more generic address 😂

Reply:
We can use this contact form: https://www.dbos.dev/contact

Reply:
The form seems too general purpose. Let us create a generic email address if we do not want use Peters email. FYI Other projects do use a committers email address

Reply:
Use `contact@dbos.dev`